src/performance/fmp.ts

```diff
@@ -137,13 +137,13 @@ class FMPTiming {
           const index: number = parseInt(item.ele.getAttribute('fmp_c'), 10);
           time = this.statusCollector[index] && this.statusCollector[index].time;
         } else {
-          const match = getStyle(item.ele, 'background-image').match(/url\(\"(.*?)\"\)/);
+          const match = getStyle(item.ele, 'background-image').match(/url\((['"]?)(.*?)\1\)/);
```

Comment:
Why do you change this? What value is it suitable for?

Reply:
background-image: url('https://www.example.com/test.png'); // single quotes
background-image: url("https://www.example.com/test.png"); // double quotes
background-image: url(https://www.example.com/test.png); // without any quotes